DOCUMENTATION.md

```diff
@@ -368,6 +368,68 @@ If the request is not successful, the client returns an `HTTPError` containing t
 |`500`|`[{`<br>`"error": "Exception",`<br>`"message": "Internal server error"`<br>`}]`|Notify was unable to process the request, resend your notification.|
 
 
+### Send a precompiled Letter
+This is an invitation-only feature. Contact the GOV.UK Notify team on the support page or through the Slack channel for more information.
+
+#### Method
+
+```python
+response = notifications_client.send_precompiled_letter_notification(
+    reference,      # Reference to identify the notification
+    pdf_file        # PDF File object
+)
+```
+
+### Arguments
+
+##### `reference` (required)
+A required identifier you generate. The `reference` can be used as a unique reference for the notification. Because Notify does not require this reference to be unique you could also use this reference to identify a batch or group of notifications.
+
```

Comment:
To bring in line with GDS / GOV.UK style: "A required identifier you create. This reference identifies a single unique notification or a batch of notifications."